### ISSUE TITLE
error when custom label

### ISSUE THREAD

**Issue by phelogges** [taki0112/StarGAN-Tensorflow#8]
When I test your pre-trained model, it always raises an error on custom label even the default label

input:
``python3 main.py --phase=test --selected_attrs=['Black_Hair','Blond_Hair','Brown_Hair','Male','Young'] --custom_label=[1,0,0,0,0]``

error:
``main.py: error: argument --custom_label: invalid int value: '[1,0,0,0,0]'``

**Comment by taki0112** [comment]
do
`python3 main.py --phase=test --selected_attrs=['Black_Hair','Blond_Hair','Brown_Hair','Male','Young'] --custom_label=1 0 0 0 0`

**Comment by phelogges** [comment]
It still did't work
error:
``main.py: error: unrecognized arguments: 0 0 0 0``

**Comment by phelogges** [comment]
quite strange, in ``main.py``, line  30:31 shows 
``parser.add_argument('--custom_label', type=int, nargs='+', help='custom label about selected attributes', default=[1, 0, 0, 0, 0])``,
which means ``[1, 0, 0, 0, 0]``is readable as default, but when I custom labels this format fails.
